packages/components/src/TooltipTrigger.ts

```diff
@@ -0,0 +1,183 @@
+import _ from 'lodash/fp';
```

Comment:
are there any important functional differences here from the original TooltipTrigger.js file other then adding TS goodness>

Reply:
TypeScript and getting colors from context vs imports. It should behave the same.